simd/unit_tests/include/SIMDTesting_Utilities.hpp

```diff
@@ -191,4 +191,27 @@ constexpr bool is_type_v = false;
 template <typename T>
 constexpr bool is_type_v<T, decltype(void(sizeof(T)))> = true;
 
+// Zero initializing each simd lane to avoid gcc's uninitialized variable
+// warnings on default constructed simd variables.
+// Using a for-loop to initialize:
+//
+// SIMDType a;
+// for(int i=0; i<SIMDType::size(); ++i)
+//   a[i] = 0;
+// return a;
+//
+// does not seem to suppress gcc's warnings.
```

Comment:
I guess that's also an artifact of the reinterpret_cast issue?

Reply:
Yes, these zero inits may no longer be needed depending on the outcome of how the reinterpret_cast issue is be handled.